packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -144,6 +149,21 @@ abstract class Route<T> {
   RouteSettings get settings => _settings;
   RouteSettings _settings;
 
+  /// The restoration scope ID to be used for the [RestorationScope] surrounding
+  /// this route.
+  ///
+  /// The restoration scope ID is null if restoration is currently disabled
+  /// for this route.
+  ///
+  /// If the restoration scope ID changes (e.g. because restoration is enabled
+  /// or disabled) throughout the life of the route, the [ValueListenable]
```

Comment:
throughout -> during

throughout implies it happening continually, during refers to a particular case of it happening